### PR TITLE
Feature/new functonality gu ivol2

### PR DESCRIPTION
**Pull request by DKusarek**
date of start sprint validation
update password
AddUsersToProject by list
user info after click on user list

### CODE REVIEW

File: ScrumIt/Forms/manageProject.cs [hunk 1]
```diff
@@ -111,10 +106,37 @@ private void editTaskButton_Click(object sender, System.EventArgs e)
...
+                    ProjectColor = ToHexValue(changeColorButton.BackColor)
+                });
+
+                var usersList = userListMenuStrip.Items;
+                var userListToAssign = new List<UserModel>();
+                foreach (ToolStripMenuItem user in usersList)
+                {
+                    if (user.Checked)
+                    {
+                        var userName = user.Name;
```
Comment (salvatien) [inline ScrumIt/Forms/manageProject.cs]: Just to make sure, user.Name is actually username (login), not firstname?

File: ScrumIt/Forms/AddSprint.cs [hunk 2]
```diff
@@ -49,6 +49,15 @@ private void addSprintButton_Click(object sender, System.EventArgs e)
                     MessageBox.Show(@"Sprint nie może być dłuższy niż 31 dni");
                     validationFlag = true;
                 }
+                else
+                {
+                    var endOfLastSprint = SprintModel.GetEndOfLastSprint(_projectId) ?? DateTime.Now.AddDays(-1);
```
[the remark this answers is not in the document]
Comment (DKusarek) [reply ScrumIt/Forms/AddSprint.cs]: I will delete part after set part after ?? to 1900 or sth, cause some date is needed to compare with other DateTime Format

File: ScrumIt/Forms/Register.cs [hunk 3]
```diff
@@ -9,15 +9,20 @@ namespace ScrumIt.Forms
...
 
         private void Register_Load(object sender, System.EventArgs e)
         {
             registeButton.BackColor = _panelColor;
+            roleComboBox.Items.Add("Wybierz rolę...");
+            roleComboBox.Items.Add("Scrum Master");
+            roleComboBox.Items.Add("Developer");
         }
 
         private void registeButton_Click(object sender, System.EventArgs e)
```
[the remark this answers is not in the document]
Comment (DKusarek) [reply ScrumIt/Forms/Register.cs]: fixed in branch feature/IntegratingUIwithBackend

File: ScrumIt/Forms/Register.cs [hunk 4]
```diff
@@ -110,6 +129,10 @@ private void userPhotoPictureBox_Click(object sender, System.EventArgs e)
                 }
             }
         }
-        
+
+        private void roleComboBox_SelectedIndexChanged(object sender, EventArgs e)
```
[the remark this answers is not in the document]
Comment (DKusarek) [reply ScrumIt/Forms/Register.cs]: fixed in branch feature/IntegratingUIwithBackend

File: ScrumIt/Forms/manageProject.cs (hunk 1 above)
Comment (DKusarek) [reply ScrumIt/Forms/manageProject.cs]: oks

File: ScrumIt/Forms/AddSprint.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (salvatien) [reply ScrumIt/Forms/AddSprint.cs]: Discuss it with Paweł, he is doing new sprint validation, maybe he wants it to stay like this to add something to it

**Pull request merged**